.github/workflows/csv-coverage-timeseries.yml

```diff
@@ -0,0 +1,44 @@
+name: Build framework coverage timeseries reports
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone self (github/codeql)
+      uses: actions/checkout@v2
+      with:
+        path: script
```

Comment:
Do you need both checkouts? Will the script work if it's run against its own checkout?

Reply:
I haven't tested this. It felt cleaner to have a working directory that's modified independently from the scripts that are generating the stats. We could probably get away with one single checkout, copying the scripts to a separate folder, and then start stepping back in the git history.

Reply:
Ok something to consider as a follow-up, doesn't have to block this PR. Checkouts of this repo are fairly quick anyway.